stdlib/py_compile.pyi

```diff
@@ -1,7 +1,5 @@
 import sys
-from typing import AnyStr, List, Optional, Text, Type, Union
-
-_EitherStr = Union[bytes, Text]
```

Comment:
Was this alias really never used?

Reply:
I suspect it was used at some point, but not anymore.